astroquery/svo_fps/core.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+import requests
+import io
+
+from astropy import units as u
+from astropy.io.votable import parse_single_table
+# The VOTables fetched from SVO contain only single table element, thus parse_single_table
+
+from ..query import BaseQuery
+
+__all__ = ['SvoFpsClass', 'SvoFps']
+
+FLOAT_MAX = np.finfo(np.float64).max
+
+
+class SvoFpsClass(BaseQuery):
+    SVO_MAIN_URL = 'http://svo2.cab.inta-csic.es/theory/fps/fps.php'
+
+    def data_from_svo(self, query, cache=True,
+                      error_msg='No data found for requested query'):
```

Comment:
I don't see this function in your documentation, do you not intend it to be used by users? If so, suggest changing to `_data_from_svo`.

Reply:
It should be documented, as it could be used by users, but as is, it's not really intended for that.  I think keeping this & documenting it properly is the right way forward.

Reply:
Sounds good. It definitely needs more documentation, and ideally an example in the the docs rst file if it's going to be public.